api/services/NorthstarService.js

```diff
@@ -40,4 +40,8 @@ module.exports = {
     return { type, id };
   },
 
+  isOneOfFieldSet(submission) {
+    return ['user_id', 'email', 'mobile'].every(userField => !submission[userField]);
+  },
+
```

Comment:
Isn't the requirement of at least one of 'user_id', 'email' or 'mobile' specific to making calls to Northstar?